ir/type.cpp

```diff
@@ -337,6 +337,10 @@ pair<expr, vector<expr>> IntType::mkInput(State &s, const char *name) const {
 }
 
 void IntType::printVal(ostream &os, State &s, const expr &e) const {
+  if (!e.isConst()) {
```

Comment:
this should never happen. What's the problem?

Reply:
This case happened when the result had a quantifier inside; `m.eval()` inside `type.printVal(os, st, m.eval(val.value, true));` failed to evaluate the result.